SharedRules.DotSettings

```diff
@@ -0,0 +1,1186 @@
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=Print/Type/@EntryValue">IdentifierPlaceholder</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=st/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=st/Properties/=Maximal/@EntryIndexedValue">-1</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=st/Properties/=Minimal/@EntryIndexedValue">-1</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=st/Type/@EntryValue">StatementPlaceholder</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/Properties/=ExactType/@EntryIndexedValue">False</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/Properties/=Type/@EntryIndexedValue"></s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/Type/@EntryValue">TypePlaceholder</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/LanguageName/@EntryValue">CSHARP</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/MatchCatchClauseWithoutExceptionFilter/@EntryValue">False</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/SearchPattern/@EntryValue">$type$ $Print$($args$)&#xD;
+{	&#xD;
+	$st$	&#xD;
+}</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/Severity/@EntryValue">SUGGESTION</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/SuppressionKey/@EntryValue">DontIncludeNumbers</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/Comment/@EntryValue">Post-fix asynchronous methods with Async of TaskAsync (SDCS-1420)</s:String>
```

Comment:
This rule does not cover cases when a method returns a regular Task.
Also, this rule will highlight code in case it is inherited from 3rd party interface (check MediatR).
**I suggest removing the patterns.**

Reply:
removed